apis/infix-en_GB/atrium-api-infix-en_GB-jvm/src/main/kotlin/ch/tutteli/atrium/api/infix/en_GB/creating/path/DirectoryEntries.kt

```diff
@@ -0,0 +1,11 @@
+package ch.tutteli.atrium.api.infix.en_GB.creating.path
+
+/**
+ *  Parameter object which collects directory entires (as [String]s).
+ *  Use the function `directoryEntry(String, vararg String)` to create this representation.
+ *
+ *  @since 0.14.0
+ */
+// TODO make inline once we drop support for Kotlin 1.2
+class DirectoryEntries(val entries: List<String>)
```

Comment:
In all other helper objects we use `T, Arra<<out T>` and extend from VarArgHelper. I would do the same here

Reply:
Done.